app/styles/currency-input.less

```
@@ -51,4 +46,12 @@
       border: 1px solid var(--color-border-default);
     }
   }
+  .currency-input.onlycurrency {
+    padding-right: 0px;
+    .currency-selector {
+      border-right: none;
+      margin-right: 0px;
+      width: 100%;
+    }
+  }
```

```suggestion
  .onlycurrency {
    padding-right: 0;
    .currency-selector {
      border-right: none;
      margin-right: 0;
      width: 100%;
    }
  }
```
Move this class in `currency-input` definition directly. You can delete `px` when you set 0. Do you need `width: 100%` here?